x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engine/engine_schema.tsx

```diff
@@ -143,12 +144,23 @@ const SchemaFieldDetails: React.FC<{ schemaField: SchemaField }> = ({ schemaFiel
 
 export const EngineSchema: React.FC = () => {
   const { engineName } = useValues(EngineIndicesLogic);
+  const [onlyShowConflicts, setOnlyShowConflicts] = useState<boolean>(false);
   const { isLoadingEngineSchema, schemaFields } = useValues(EngineViewLogic);
   const { fetchEngineSchema } = useActions(EngineViewLogic);
   const [itemIdToExpandedRowMap, setItemIdToExpandedRowMap] = useState<Record<string, JSX.Element>>(
     {}
   );
 
+  const toggleOnlyShowConflicts = useCallback(() => {
+    setOnlyShowConflicts(!onlyShowConflicts);
+    setItemIdToExpandedRowMap({});
+  }, [setItemIdToExpandedRowMap, onlyShowConflicts]);
+
+  const filteredSchemaFields = useMemo(() => {
+    if (!onlyShowConflicts) return schemaFields;
```

Comment:
nitpick(non-blocking): checking the negative of a boolean is hard to understand. Could we flip these so it's:
```
if (onlyShowConflicts) return schemaFields.filter((field) => field.type === 'conflict');
return schemaFields;
```

Reply:
✅ https://github.com/elastic/kibana/pull/154149/commits/e08b77964771f0ab82fb1f797d278e8d5eb8e3cf